lmdeploy/pytorch/engine/executor/base.py

```diff
@@ -27,6 +27,9 @@ def __init__(self,
                  device_type: str = 'cuda'):
         """Initialize Executor."""
         cache_config.window_size = model_config.sliding_window
+        if cache_config.window_size is not None and cache_config.window_size > 0:
+            # do not support sliding window prefix caching
```

Comment:
it would be better to give the user a warning message if enable_prefix_caching is explicitly set to True.